service/src/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -499,6 +592,8 @@ public TExecuteStatementResp ExecuteStatement(TExecuteStatementReq req) throws T
     try {
       SessionHandle sessionHandle = new SessionHandle(req.getSessionHandle());
       String statement = req.getStatement();
+      startFunction(ThriftCliFunctions.ExecuteStatement,
+          "\tstmt={" + statement.replaceAll("[\\r\\n\\t]", "") + "}");
```

Comment:
The `if(auditLog.isInfoEnabled())` call is good `logAuditEvent` but it does not prevent this code from running, these replaceAll operations or other string operations are all unnecessary. I'm not sure these would be optimized out by itself.